src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/util/ManagedIndexUtils.kt

```diff
@@ -330,7 +330,8 @@ fun ManagedIndexMetaData.getStartingManagedIndexMetaData(
     return this.copy(
         stateMetaData = updatedStateMetaData,
         actionMetaData = updatedActionMetaData,
-        stepMetaData = updatedStepMetaData
+        stepMetaData = updatedStepMetaData,
+        info = mapOf("message" to "Starting action ${action.type} and working on ${step.name}")
```

Comment:
We may want to avoid referencing ```step``` information in the ```info``` for ```ManagedIndexMetaData```

Reply:
right I tried to avoid having step information so worded such that "action X is working on Y" so it still doesn't have notion of a "Step". 

Any other suggestion on how we can differentiate the info without giving too much about the "Step"?

Reply:
I think this is fine then.